lib/appium.js

```diff
@@ -69,6 +70,11 @@ class AppiumDriver extends BaseDriver {
    }
 
   async deleteSession (sessionId) {
+    if (!this.sessions[sessionId]) {
```

Comment:
we should never run into this case, because `deleteSession` should be guarded by `sessionExists`. if this is not the case, it's either a bug here, in base-driver, or mjsonwp